pydantic/main.py

```diff
@@ -307,6 +307,7 @@ def dict(
         exclude_unset: bool = False,
         exclude_defaults: bool = False,
         exclude_none: bool = False,
+        exclude_private: bool = True,
```

Comment:
remove this, I think it's safe to always exclude anything in `__dict__` where the key startswith with `_`.

Reply:
But this would break concept when we have only parsed values in `__dict__` and will force additional checks, etc.

Reply:
```py
from pydantic import BaseModel, Extra

class Record(BaseModel):
    class Config:
        extra = Extra.allow

print(Record(_a='b'))  # _a='b'
```